engine/src/main/java/org/hibernate/validator/messageinterpolation/ParameterMessageInterpolator.java

```diff
@@ -52,7 +52,7 @@ public ParameterMessageInterpolator(Set<Locale> locales, Locale defaultLocale, L
 	}
 
 	@Override
-	public String interpolate(Context context, Locale locale, String term) {
+	protected String interpolate(Context context, Locale locale, String term) {
```

Comment:
Isn't this technically an API change? I'm not sure it will impact anyone, but still...

Reply:
It is but I think it was a mistake.

One information you might miss is I won't merge this to 6.1, I will do a 6.2 and also include this in 7.